R/forward_geocoding.R

```diff
@@ -69,6 +69,38 @@ oc_forward <-
     oc_parse(res)
   }
 
+#' @export
+oc_forward_df <-
+  function(placename,
+           key = oc_key(),
+           bounds = NULL,
+           countrycode = NULL,
+           language = NULL,
+           limit = 10,
+           min_confidence = NULL,
+           no_annotations = FALSE,
+           no_dedupe = FALSE,
+           no_record = FALSE,
+           abbrv = FALSE,
+           add_request = TRUE) {
```

Comment:
The request is missing in the output. That is probably not a problem though, if we `bind_cols` the  source_df and the results_df (as discussed [here](https://github.com/ropensci/opencage/pull/31#issuecomment-365442732).  We'd have to make sure though, that the source_df and the results_df are of equal length, of course. Otherwise we would have to join, which can get messy if there are duplicate addresses.

Reply:
I have worked beyond this pull request and created an output option that uses `bind_cols` and forces the limit to 1, because I think that has to be the most useful output. This gives output options for either returning multiple results with a query column or one result and combining the input data frame and the results data frame.